packages/genui/a2ui-playground/lynx-src/a2ui/index.css

```
@@ -15,6 +15,37 @@
 
+.a2ui-container {
+  width: 100%;
+}
+
+.a2ui-scrollPaused .a2ui-container {
+  opacity: 0.92;
+}
+
+.a2ui-loadingOverlay {
+  position: absolute;
+  display: flex;
```

Stylelint (17.11.0)

[error] Duplicate property "display" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)